suite_held_suarez_1994.xml

```diff
@@ -7,5 +7,9 @@
     <scheme>apply_tendency_of_northward_wind</scheme>
     <scheme>apply_heating_rate</scheme>
     <scheme>geopotential_temp</scheme>
+    <scheme>cam_state_diagnostics</scheme>
+  </group>
+  <group name="physics_after_coupler">
+    <scheme>cam_tend_diagnostics</scheme>
```

Comment:
Can you add this to the `suite_cam7.xml` file as well?

Reply:
added!